webui/react/src/components/HyperparameterSearchModal.module.scss

```diff
@@ -14,7 +14,7 @@
     :global(.ant-modal-confirm-btns) {
       display: none;
     }
-    :global(.ant-form-item-explain) {
+    div[class*='ant-form-item-explain'] {
```

Comment:
This doesn't fix the issue. Error messages on non hyperparameter search modals are still hidden.